src/containers/Network/test/validators.test.js

```diff
@@ -24,7 +24,7 @@ const WS_URL = 'ws://localhost:1234';
 
 describe('Validators Tab container', () => {
   let server;
-  let client;
+  let client = new MockWsClient(WS_URL);
```

Comment:
It's defined in the `beforeEach`, it doesn't need to be defined up here.

Reply:
Removed.  It had been throwing errors for me at one point saying it was undefined but that was before some of your changes.